lab-trent/test/tcp-test.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const assert = require('assert');
+const net = require('net');
+const client = new net.Socket();
+
+require('../index');
+
+describe('Client', function() {
+  beforeEach(function(done) {
+    client.connect( { host: '127.0.0.1', port: 43593 }, function() {
+      done();
+    });
+  });
+
+  afterEach(function(done) {
+    client.destroy();
```

Comment:
Probably best to use `client.end()` rather than destroy. End will allow any final packets to be received prior to destroying the client, where destroy will end without any regard to remaining data in the stream.